inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt

```diff
@@ -72,11 +96,15 @@ class MainActivity : AppCompatActivity() {
         var switch = view as Switch
         if (switch.isChecked) {
             inkView.dynamicPaintHandler = FancyPaintHandler()
+            // inkView.dynamicPaintHandler = HighlighterPaintHandler()
```

Comment:
do we need this here? if its blog related, maybe add a comment to explain

Reply:
removed!